pkg/operator/operator/k8s.go

```diff
@@ -578,20 +579,41 @@ func pythonDownloadArgs(api *spec.API) string {
 }
 
 func onnxDownloadArgs(api *spec.API) string {
-	downloadConfig := downloadContainerConfig{
-		LastLog: fmt.Sprintf(_downloaderLastLog, "onnx"),
-		DownloadArgs: []downloadContainerArg{
-			{
-				From:             config.BucketPath(api.ProjectKey),
-				To:               path.Join(_emptyDirMountPath, "project"),
-				Unzip:            true,
-				ItemName:         "the project code",
-				HideFromLog:      true,
-				HideUnzippingLog: true,
-			},
+	downloadContainerArs := []downloadContainerArg{
+		{
+			From:             config.BucketPath(api.ProjectKey),
+			To:               path.Join(_emptyDirMountPath, "project"),
+			Unzip:            true,
+			ItemName:         "the project code",
+			HideFromLog:      true,
+			HideUnzippingLog: true,
 		},
 	}
 
+	if api.Predictor.Models.Path != nil && strings.HasSuffix(*api.Predictor.Models.Path, ".onnx") {
+		downloadContainerArs = append(downloadContainerArs, downloadContainerArg{
+			From: *api.Predictor.Models.Path,
+			To:   path.Join(_modelDir, consts.SingleModelName, "1"),
```

Comment:
nit: I think if we add `ItemName: "the onnx model"` to the struct, it will print a nice message in the downloader container

Reply:
Thought of not doing it because adding this log would have been misleading since we advertise the `predictor.models` predictor as having the LR active. That being said, I actually think this is a good idea because it highlights to the user that a model gets downloaded at deploy time (suggestive that it's not LR) and it's also potentially easier for us to debug if the user shares their logs.